.travis.yml

```diff
@@ -54,7 +54,7 @@ after_success:
         "develop")
           export APP_INSTANCE="dev";
           ;;
-        "feature/vue")
+        "f/class-list")
```

Comment:
This probably should be changed back to `feature/vue`

Reply:
Definitely, updated.